src/summersplash2022/sections/søkJobb.tsx

```diff
@@ -1,12 +1,23 @@
 import style from './søkJobb.module.css';
+import sharedStyle from '../index.module.css';
 import { ApplyType } from 'src/summersplash2022/utils/utils';
 import ApplyImgWithBlob from 'src/summersplash2022/img/applyImgWithBlob';
 
 const Apply = ({ applyType }: { applyType: ApplyType }) => {
   const isApplyGraduate = applyType == ApplyType.GRADUATE;
+  const variantImg = require(isApplyGraduate
+    ? 'src/components/page-header/whiteVariant.svg'
+    : 'src/components/page-header/variant.svg');
 
   return (
-    <section className={style.section} id="sokfastjobb">
+    <section
+      className={`${style.section} ${
+        isApplyGraduate
+          ? sharedStyle.sectionDarkTeal
+          : sharedStyle.sectionLightPeach
+      }`}
+      id="sokfastjobb"
```

Comment:
Om denne skal gjenbrukes burde kanskje de søk sommerjobb/fastjobb knappene begge lenke til samme anchor? Denne kan endres til f.eks 
```suggestion
      id="sokjobb"
```

og da la begge de knappene på /sommerjobb og /nyutdannet lenke til den

Reply:
(nå så er i alle fall knappen `Søk sommerjobb` på `/sommerjobb` ikke-fungerende)